src/main/java/duke/Echo.java

```diff
@@ -0,0 +1,26 @@
+package duke;
+
+/**
+ * The Echo class is used to automatically format what Duke says.
+ * This is the other half of the analogue to the 'Ui' class in the example.
+ * Anytime you see Echo, it's UI.
+ *
+ * @author Rdac0
+ */
```

Comment:
Shouldn't the author tag be your name rather than your Github username? 
Maybe something like @author "Your Full Name" (Github username) will be better?